File: Day 07 - Arrays/solution.js
function main() {
  const n = parseInt(readLine(), 10);

  const arr = readLine()
    .split(" ")
    .map(arrTemp => parseInt(arrTemp, 10));
  var aux = "";
  for (var i = n - 1; i >= 0; i--) {
    aux = aux + arr[i] + " ";
  }
  console.log(aux);
}

File: Day 07 - Arrays/solution.py
n = int(input())
arr = list(map(int, input().rstrip().split()))

reverse = ""
for i in range(n-1, -1, -1):
    reverse += str(arr[i]) + " "
print(reverse)

# Option 2
# aux = ""
# for i in range(len(arr))[::-1]:
#   aux = aux + str(arr[i]) + " "
# print(aux)

# Option 3
# print(" ".join(map(str, arr[::-1])))
